tests/system/test-api-search-post.js

```diff
@@ -329,10 +336,10 @@ test('/search preserve intersects geometry in next link', async (t) => {
   let response = await t.context.api.client.post('search', {
     json: {
       intersects: intersectsGeometry,
-      limit: 2
+      limit: 3
     }
   })
-  t.is(response.features.length, 2)
+  t.is(response.features.length, 3)
   t.is(response.links.length, 0)
```

Comment:
the fact that links length was 0 here was because the collections_2 item did not match the query because the index was setup incorrectly and the geometry was not a geo_shape.